web/src/components/clustering/TimeSeries.vue

```diff
@@ -183,3 +244,18 @@ onUnmounted(() => {
   simulation.value?.stop();
 });
 </script>
+
+<style lang="scss">
+.timeseries {
+  position: relative;
+
+  &-node {
+    cursor: v-bind("nodeCursor");
+
+    &.selected {
+      stroke: red;
```

Comment:
The red stroke is difficult to spot for nodes filled in red-ish colors, and it is also visually not great, clashing with the D3 color palette.
- We could use a black stroke to highlight the selected node, but this will be confusing with how we highlight the first submitter on the graph page.
- As an alternative, we can drop the saturation of other nodes, like we already do on the graph page. But this is not ideal for colorblindness.
- Having a concentric circle around it (stroke around an invisible larger circle), will be visually the best I think, but is technically a bit more cumbersome. This could either be achieved by using stroke gradients (hacky) or putting nodes in `g`roups in the svg.

Reply:
Right now there is no first submitter highlighting on the graph, as this was an issue that needed some brainstorming.
I will change this to black for now.